theano/gpuarray/subtensor.py

```diff
@@ -544,6 +544,12 @@ def perform(self, node, inputs, out_):
         idx_ = ([slice(None)] * p + nidx[p:])
         x = x.__getitem__(idx_)
 
+        if p == 0:
+            # The only indexing was through slices and indices.
+            # This can happen with symbolic slices for instance.
+            out[0] = x
```

Comment:
Good point, that would actually be in place.
Since this case can be detected in advance, we could set the view_map, but I think for now I'll just force a copy.